core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -814,25 +890,23 @@ private MasterWorkerInfo findUnregisteredWorker(long workerId) {
    * @param workerId the worker id to register
    */
   @Nullable
-  private MasterWorkerInfo registerWorkerInternal(long workerId) {
+  private MasterWorkerInfo recordWorkerRegistration(long workerId) {
```

Comment:
This method just updates the `DefaultBlockMaster` internal worker list, adding this worker to the internal available worker list. This methods only needs 2 fields from the `MasterWorkerInfo`:
1. `worker.getAddress()` which is a final field
2. `worker.updateLastUpdatedTimeMs()` which is removed from this method(see comment below).

Therefore this methods does not need locking.